Src/Support/Google.Apis.Auth.Tests/OAuth2/ComputeCredentialTests.cs

```diff
@@ -44,52 +45,63 @@ public void IsRunningOnComputeEngine_ResultIsCached()
         [Fact]
         public async Task FetchesOidcToken()
         {
-            var clock = new MockClock { UtcNow = new DateTime(2020, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc) };
-            var messageHandler = new OidcTokenSuccessMessageHandler(clock);
+            // A little bit after the tokens returned from OidcTokenFakes were issued.
+            var clock = new MockClock { UtcNow = new DateTime(2020, 5, 13, 15, 0, 0, 0, DateTimeKind.Utc) };
+            var messageHandler = new OidcTokenSuccessMessageHandler();
             var initializer = new ComputeCredential.Initializer("http://will.be.ignored", "http://will.be.ignored")
             {
                 Clock = clock,
                 HttpClientFactory = new MockHttpClientFactory(messageHandler)
             };
             var credential = new ComputeCredential(initializer);
 
-            var oidcToken = await credential.GetOidcTokenAsync(OidcTokenOptions.FromTargetAudience("audience"));
+            // The fake Oidc server returns valid expired tokens but with a set audience
```

Comment:
I'd expect the token to *not* be expired. The documentation says it was issued "May 13, 2020 from 2:54pm UTC and for 1 hour."
We're within that range, so it should be okay, right?

Reply:
(Yes, it's expired in the real world - but in the context of the test, it's not expired.)